package.json

```diff
@@ -16,6 +16,7 @@
     "lint": "eslint .",
     "prepublishOnly": "yarn build",
     "prebuild": "rm -rf dist",
+    "postinstall": "yarn build",
```

Comment:
As far as I know - postinstall will be executed on a client, without any dev deps, it actually needs to be executed.
Should not work.